DESCRIPTION

```diff
@@ -57,6 +57,7 @@ Suggests:
     covr,
     devtools,
     knitr,
+    nmrec,
```

Comment:
Maybe there's a reason you didn't do this, but we need to add a repo for `nmrec` in the `pkgr.yml` for our dev package install directions to work.

Reply:
Good catch.  I know we talked about it and agreed it was fine to do; I just thought I had done it :/